jenkins/jobs/glusterfs-rpms.yml

```diff
@@ -0,0 +1,51 @@
+- job:
+    name: glusterfs-rpms-test
+    node: glusterfs-devrpms
+    description: Creates RPMs for developers and those who need to verify the patch
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        branches:
+        - $GERRIT_BRANCH
+        refspec: $GERRIT_REFSPEC
+        url: http://review.gluster.org/glusterfs
+
+    triggers:
+      - gerrit:
+          trigger-on:
+            - change-merged-event
+          projects:
+            - project-compare-type: 'PLAIN'
+              project-pattern: 'glusterfs'
+              branches:
+                - branch-compare-type: 'PLAIN'
+                  branch-pattern: 'master'
+
+    builders:
+    - shell: CFGS="fedora-22-x86_64" /opt/qa/jenkins/scripts/glusterfs-rpms.sh
+
+    logrotate:
+      artifactDaysToKeep: '10'
+      artifactNumToKeep: '10'
+      daysToKeep: '10'
+      numToKeep: '10'
+
+    parameters:
+    - string:
+        default: refs/heads/master
+        description: ''
+        name: GERRIT_REFSPEC
+    - string:
+        default: master
+        description: ''
+        name: GERRIT_BRANCH
+
+    publishers:
+    - archive:
+        allow-empty: false
+        artifacts: RPMS/**
```

Comment:
So I didn't look at the job, but that include RPMS and SRPMS ? (since that would be the easiest way to fullfill GPL... )